### PR TITLE
[NodeSearchBundle] Added tag for node searcher services

### PR DESCRIPTION
**Pull request by acrobat**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | 

Add a tag for node searcher services to deprecate using the container to directly get the search service during a search

### CODE REVIEW

**Review by ProfessorKuma — APPROVED** [review]
Hi @acrobat, your PR passed all our requirements.

Thank you for contributing!

**Review by ProfessorKuma — APPROVED** [review]
Hi @acrobat, your PR passed all our requirements.

Thank you for contributing!

**Review by ProfessorKuma — APPROVED** [review]
Hi @acrobat, your PR passed all our requirements.

Thank you for contributing!

**Review by ProfessorKuma — APPROVED** [review]
Hi @acrobat, your PR passed all our requirements.

Thank you for contributing!

**Review by ProfessorKuma — APPROVED** [review]
Hi @acrobat, your PR passed all our requirements.

Thank you for contributing!

**Review by ProfessorKuma — APPROVED** [review]
Hi @acrobat, your PR passed all our requirements.

Thank you for contributing!

**Review by ProfessorKuma — APPROVED** [review]
Hi @acrobat, your PR passed all our requirements.

Thank you for contributing!

File: UPGRADE-5.2.md [hunk 1]
```diff
@@ -9,9 +9,15 @@ General
...
 
 * We've removed the `RoleInterface` on the `Kunstmaan\AdminBundle\Entity\Group` entity if you run your code on symfony 4. 
   The interface was deprecated and removed in symfony 4. If you used this interface to check the `Group` entity change it to
   the `FOS\UserBundle\Model\GroupInterface`. The `Group` entity won't change if you run on symfony 3.4 but it's adviced to make 
   this change already.
+
+NodeSearchBundle
+----------------
+
+* Depending on the service container to retreive searchers is deprecated and will be removed in 6.0. Tag your custom node
```
Comment (Numkil) [inline UPGRADE-5.2.md]: retrieve

**Review by Numkil — APPROVED** [review]

**Review by ProfessorKuma — APPROVED** [review]
Hi @acrobat, your PR passed all our requirements.

Thank you for contributing!

File: UPGRADE-5.2.md (hunk 1 above)
Comment (acrobat) [reply UPGRADE-5.2.md]: Fixed!

**Pull request merged**